framework/wazuh/core/cluster/master.py

```diff
@@ -113,12 +113,13 @@ def __init__(self, **kwargs):
         self.sync_integrity_free = True  # the worker isn't currently synchronizing integrity
         self.sync_extra_valid_free = True
         # Sync status variables. Used in cluster_control -i and GET/cluster/healthcheck.
-        self.sync_integrity_status = {'date_start_master': "n/a", 'date_end_master': "n/a",
+        self.integrity_check_status = {'date_start_master': "n/a", 'date_end_master': "n/a"}
```

Comment:
Use default values of the same type as the values to be operated later on. (datetime, int, etc)